src/templates/ml-facets-inline.html

```diff
@@ -4,8 +4,10 @@
     <div ng-repeat="value in facet.facetValues" ng-class="{ 'on': value.selected} ">
       <a ng-click="toggle({facet: facet.__key, value: value.name})" title="{{ value.name }}">
         <i ng-class="{ 'fa fa-trash-o': value.selected }"></i>
-        {{ value.name | truncate:20 }} ({{ value.count }})
+        <span ng-if="value.name">{{ value.name | truncate:truncateLength }}</span>
```

Comment:
The `truncate` filter appends an "end" string, if the input is greater than `truncateLength`.

In any case, the truncate filter will be included for backward compatibility. Most (if not all) demos using this project depend on it.